### DISCUSSION TITLE
Q & A about Private Key

### DISCUSSION THREAD

**Discussion by HalloKyle** [IBAX-io/go-ibax#62] [Weaver Wallet]
![privatekey](https://user-images.githubusercontent.com/51123970/149915791-f52f90be-9699-4f3f-8609-25698796b6b1.png)

**Comment by HalloKyle** [comment]
The password is used to encrypt your private key and allow easy access to your account
IBAX does not store your private key, so the private key associated with the password will be stored in your browser's cookie.
When you clear the cookie or use a new browser to access your IBAX account, you will need to import your helper notes and encrypt them again with your password
Of course you can also regenerate an IBAX account and re-encrypt it with a password

hint：

- Take care of your private key (account seed)
- Take care of your public key
- Take care of your password
- Take care of your wallet address

**Reply by ngay12** [reply]
i only save phrase but not private key how open using phrase?

**Reply by cohotho** [reply]
Thanks you

**Reply by harunkursun23** [reply]
HEPSİNİ KAYDETTİM BEN NE OLUR NE OLMAZ DİYE
İŞİMİ ŞANSA BIRAKMAYI SEVMİYORUM PEK

**Reply by ibrahimclskn66** [reply]
Good project

**Reply by hizen1984** [reply]
many people had problems with the wallet, thanks for the information, nice job

**Reply by samet010114** [reply]
teşekkürler

**Reply by ZagorKahraman** [reply]
> Parola özel anahtarınızı şifrelemek ve hesabınıza kolay erişim sağlamak için kullanılır IBAX özel anahtarınızı saklamaz, bu nedenle parolayla ilişkili özel anahtar tarayıcınızın tanımlama bilgisinde saklanır. Çerezleri temizlediğinizde veya IBAX hesabınıza erişmek için yeni bir tarayıcı kullandığınızda, yardımcı notlarınızı içe aktarmanız ve bunları şifrenizle tekrar şifrelemeniz gerekecektir. Tabii ki bir IBAX hesabını yeniden oluşturabilir ve bir şifre ile yeniden şifreleyebilirsiniz.
> 
> ipucu:
> 
> * Özel anahtarınıza dikkat edin (hesap tohumu)
> * Genel anahtarınıza dikkat edin
> * Parolanıza dikkat edin
> * Cüzdan adresinize dikkat edin

Eyw

**Reply by jedac** [reply]
Çok insan patlayacak bu cüzdan yüzünden

**Reply by CASPERLAND** [reply]
evet

**Reply by Sehoo72** [reply]
Teşekkürler dostum. Eline sağlık

**Reply by coinnapolyon** [reply]
When i want to access my account system wants Private Key not Account Seed. They are not same. How can i find myy Private Key ?

**Comment by HalloKyle** [comment]
Anyone here?

**Reply by Alevyilmaz21** [reply]
Var😂

**Reply by junbujian666** [reply]
good

**Comment by coinsinov** [comment]
Thank You

**Reply by Mdnurunnabi1** [reply]
Good

**Comment by ngay12** [comment]
thanks i made it

**Reply by Mdnurunnabi1** [reply]
Hi

**Comment by Turbolaz61** [comment]
Hehe sizin köy daha büyük

**Reply by Haseki2162** [reply]
👍

**Comment by revere23** [comment]
p

**Comment by sedat2855** [comment]
THX

**Reply by Mdnurunnabi1** [reply]
Hi

**Comment by Trajedi032** [comment]
Benim kaydettiğim kod ve şifre hata veriyor nede

**Comment by mustafa0505** [comment]
thank you

**Comment by Alevyilmaz21** [comment]
Cüzdan adresimizi kaybedersek cüzdanı yeniden kurtarabilirmiyiz

**Reply by kac4r** [reply]
Hayir

**Comment by mustafa0505** [comment]
Kaydolurken çıkan kelimeleri tekrar girmek gerekir diye biliyorum. Veya
tekrar kayıt olmayı deneyin. Tekrar kayıt sorun olmamalı.

29 Oca 2022 Cmt 22:22 tarihinde Alevyilmaz21 ***@***.***>
şunu yazdı:

**Reply by Alevyilmaz21** [reply]
Tesekur ederim 🙏

**Comment by Endijuan33** [comment]
for now the UI is very unfriendly, maybe it would be better if it was improved again to make it easier to use

**Comment by MMaraung** [comment]
great project

**Comment by hizen1984** [comment]
very good project
________________________________
Gönderen: CASPERLAND ***@***.***>
Gönderildi: 30 Ocak 2022 Pazar 18:35
Kime: IBAX-io/go-ibax ***@***.***>
Bilgi: hizen1984 ***@***.***>; Comment ***@***.***>
Konu: Re: [IBAX-io/go-ibax] Q & A about Private Key (Discussion #62)


evet

—
Reply to this email directly, view it on GitHub<https://github.com/IBAX-io/go-ibax/discussions/62#discussioncomment-2076476>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/AXOP657YCYDR4V5Z5XLDQUDUYVLFJANCNFSM5MGXKOHA>.
Triage notifications on the go with GitHub Mobile for iOS<https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675> or Android<https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.

**Comment by serdarkokturk** [comment]
Good

**Comment by Sehoo72** [comment]
Süper

**Comment by Alikopterli517** [comment]
İbax cüzdanı için uygulama kesinlikle olmalı

**Comment by muhammedonen** [comment]
Future project 👏👏👏
